src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +55,9 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+	@Override
+	public String getPrintableString() {
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.